lineman/app/components/thread_page/proposal_positions_panel/proposal_positions_panel.coffee

```diff
@@ -5,6 +5,8 @@ angular.module('loomioApp').directive 'proposalPositionsPanel', ->
   replace: true
   controller: ($scope, Records, CurrentUser, ModalService, VoteForm) ->
 
+    $scope.undecidedPanelOpen = false
```

Comment:
Not necessary but makes it easier to say "is false or true" rather than "is undefined or false or true"